src/tactics/Board.js

```diff
@@ -2007,6 +2012,20 @@ export default class Board {
       }
     });
   }
+  isAdjacentToHighlighted({ x, y }, includeDiagonal = true) {
+    return [
+        [x - 1, y],
+        [x + 1, y],
+        [x, y - 1],
+        [x, y + 1]
+      ].concat(!includeDiagonal ? [] : [
+        [x - 1, y - 1],
+        [x - 1, y + 1],
+        [x + 1, y - 1],
+        [x + 1, y + 1]
+      ]).map(([x,y]) => this.getTile(x,y))
+      .find(t => this._highlighted.get(t));
```

Comment:
getting rid of the map and changing the find to 

```js
.find(([x,y]) => this._highlighted.get(this.getTile(x, y)));
```
would save on `getTile` calls.

Reply:
Good point.  Please proceed with this change.